charts/attributes/templates/configmap.yaml

```
@@ -13,8 +13,8 @@ data:
   OIDC_TOKEN_URL: {{ ( include "attributes.oidc.externalUrl" . ) }}/realms/{{ .Values.oidc.realm}}/protocol/openid-connect/token
   OIDC_SCOPES: {{ .Values.oidc.scopes | quote  }}
   OPENAPI_URL: {{ .Values.openapiUrl | quote }}
-  POSTGRES_DATABASE: {{ .Values.postgres.database | quote  }}
-  POSTGRES_HOST: {{ .Values.postgres.host | quote  }}
+  POSTGRES_DATABASE: {{ .Values.global.opentdf.common.postgres.database | quote  }}
```

Does something like this work? 

```suggestion
  POSTGRES_DATABASE: {{ .Values.postgres.host | default .Values.global.opentdf.common.postgres.database | quote  }}
```

This would allow existing chart usages (non-backend) to continue working without changing their values, and would also allow per-micro Postgres selection.

The discussion at https://stackoverflow.com/questions/59250132/helm-using-global-values-by-default provides conflicting reports. I think you may have to add empty (falsy) values to these back in the values files.

If you want to go ahead with this that should be okay as we aren't planning a release at the moment but I may go back in and add it later.

---

An approach like this could work where you don't have to define all the values if the if condition is ordered correctly:
https://itnext.io/helm-3-umbrella-charts-global-values-in-sub-charts-666437d4ed28